modules/installation-gcp-permissions.adoc

```diff
@@ -29,6 +29,267 @@ account requires the following permissions. If you deploy your cluster into an e
+* `compute.subnetworks.use`
+* `compute.subnetworks.useExternalIp`
+====
+
+.Required permissions for creating load balancer resources
+[%collapsible]
+====
+* `compute.regionBackendServices.create`
+* `compute.regionBackendServices.get`
+* `compute.regionBackendServices.list`
+* `compute.regionBackendServices.update`
+* `compute.regionBackendServices.use`
+* `compute.targetPools.addInstance`
+* `compute.targetPools.create`
+* `compute.targetPools.get`
+* `compute.targetPools.list`
+* `compute.targetPools.removeInstance`
+* `compute.targetPools.use`
+====
+
+.Required permissions for creating DNS resources
+[%collapsible]
+====
+* `dns.changes.get`
+* `dns.changes.create`
+* `dns.managedZones.create`
+* `dns.managedZones.get`
+* `dns.managedZones.list`
+* `dns.networks.bindPrivateDNSZone`
+* `dns.resourceRecordSets.create`
+* `dns.resourceRecordSets.list`
+====
+
+.Required permissions for creating Service Account resources
+[%collapsible]
+====
+* `iam.serviceAccountKeys.create`
+* `iam.serviceAccountKeys.delete`
+* `iam.serviceAccountKeys.get`
+* `iam.serviceAccountKeys.list`
+* `iam.serviceAccounts.actAs`
+* `iam.serviceAccounts.create`
+* `iam.serviceAccounts.delete`
+* `iam.serviceAccounts.get`
+* `iam.serviceAccounts.list`
+* `resourcemanager.projects.get`
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+====
+
+.Required permissions for creating compute resources
+[%collapsible]
+====
+* `compute.disks.create`
+* `compute.disks.get`
+* `compute.disks.list`
+* `compute.instanceGroups.create`
+* `compute.instanceGroups.delete`
+* `compute.instanceGroups.get`
+* `compute.instanceGroups.list`
+* `compute.instanceGroups.update`
+* `compute.instanceGroups.use`
+* `compute.instances.create`
+* `compute.instances.delete`
+* `compute.instances.get`
+* `compute.instances.list`
+* `compute.instances.setLabels`
+* `compute.instances.setMetadata`
+* `compute.instances.setServiceAccount`
+* `compute.instances.setTags`
+* `compute.instances.use`
+* `compute.machineTypes.get`
+* `compute.machineTypes.list`
+====
+
+.Required for creating storage resources
+[%collapsible]
+====
+* `storage.buckets.create`
+* `storage.buckets.get`
+* `storage.buckets.delete`
+* `storage.buckets.list`
+* `storage.objects.create`
+* `storage.objects.delete`
+* `storage.objects.get`
+* `storage.objects.list`
+====
+
+.Required permissions for creating health check resources
+[%collapsible]
+====
+* `compute.healthChecks.create`
+* `compute.healthChecks.get`
+* `compute.healthChecks.list`
+* `compute.healthChecks.useReadOnly`
+* `compute.httpHealthChecks.create`
+* `compute.httpHealthChecks.get`
+* `compute.httpHealthChecks.list`
+* `compute.httpHealthChecks.useReadOnly`
+====
+
+.Required permissions to get GCP zone and region related information
+[%collapsible]
+====
+* `compute.globalOperations.get`
+* `compute.regions.list`
+* `compute.regionOperations.get`
+* `compute.zones.get`
+* `compute.zones.list`
+* `compute.zoneOperations.get`
```

Comment:
Suggest to rearrange the permissions in ascending order as below
* `compute.globalOperations.get`
* `compute.regionOperations.get`
* `compute.regions.list`
* `compute.zoneOperations.get`
* `compute.zones.get`
* `compute.zones.list`

Reply:
Addressed